app/components/post_body/post_body.js

```diff
@@ -104,6 +118,10 @@ export default class PostBody extends PureComponent {
                 isLongPost: true,
             });
         }
+
+        if (this.props.isLastPost) {
+            this.logTelemetry();
```

Comment:
Though last post doesn't mean last to render, this makes a good initial indicator on when to end capturing of metrics like overall load time.